candi/bashible/bashible.sh.tpl

```diff
@@ -253,6 +253,16 @@ function main() {
     else
       REBOOT_ANNOTATION=null
   fi
+ if [ "$FIRST_BASHIBLE_RUN" != "yes" ]; then
+    while true; do
```

Comment:
let to move this cycle into bb-functions, for example in file https://github.com/deckhouse/deckhouse/blob/main/candi/bashible/bashbooster/60_deckhouse.sh

Because we use same logic in more places
https://github.com/deckhouse/deckhouse/pull/14828/files#diff-2d8333a5fdf6d09d17b9afa563be709bf2e40738abdc6a2a2e871c9a15d7a704R42

https://github.com/deckhouse/deckhouse/pull/14828/files#diff-2d8333a5fdf6d09d17b9afa563be709bf2e40738abdc6a2a2e871c9a15d7a704R22